### PR TITLE
Update dockerfiles for forecast jobs

### PR DESCRIPTION
**Pull request by Mnoble-19**
**_WHAT DOES THIS PR DO?_**
- Combines forecast job & training stages into one.

### CODE REVIEW

File: src/predict/jobs/forecast/Dockerfile [hunk 1]
```diff
@@ -10,30 +10,18 @@ WORKDIR /app/
 RUN pip install --upgrade pip
 RUN pip install -r requirements.txt
 
-# Production - daily forecasts
-FROM base as production_daily
+# Production
+FROM base as production
 ENV ENV=production
-CMD ["python", "daily_forecasts.py"]
+CMD ["sh", "run_scripts.sh"]
```
Comment (Mnoble-19) [inline src/predict/jobs/forecast/Dockerfile:16]: @123MwanjeMike . I tried to have 2 CMDs for each script but got the warning. `There can only be one CMD instruction in a Dockerfile or build stage. Only the last one will have an effect` so I've just used a bash script. Don't know why I didn't consider that first. 😅🤡🤦🏾‍♂️

Comment (123MwanjeMike) [reply src/predict/jobs/forecast/Dockerfile:16]: Thanks for this improvement @Mnoble-19 
For the 3-5minutes timespan, the VMs run for 8minutes by default so we are good.

**Review by 123MwanjeMike — APPROVED** [review]
This is a great improvement.
Thanks @Mnoble-19

**Review by 123MwanjeMike — CHANGES_REQUESTED** [review]
Hi @Mnoble-19 
I'm seeing the error log below failing the deploy preview. Could we have it fixed too?

image

**Review by 123MwanjeMike — APPROVED** [review]

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Pull request merged**